src/views/PropertyRegistration.vue

```diff
@@ -130,112 +145,94 @@
-          this.state = "CE"
-
-        } else if (this.state == "Distrito Federal") {
-          this.state = "DF"
-
-        } else if (this.state == "Espírito Santo") {
-          this.state = "ES"
-
-        } else if (this.state == "Goiás") {
-          this.state = "GO"
-
-        } else if (this.state == "Maranhão") {
-          this.state = "MA"
-
-        } else if (this.state == "Mato Grosso") {
-          this.state = "MT"
-
-        } else if (this.state == "Mato Grosso do Sul") {
-          this.state = "MS"
-
-        } else if (this.state == "Minas Gerais") {
-          this.state = "MG"
-
-        } else if (this.state == "Pará") {
-          this.state = "PA"
-
-        } else if (this.state == "Paraíba") {
-          this.state = "PB"
-
-        } else if (this.state == "Paraná") {
-          this.state = "PR"
-
-        } else if (this.state == "Pernambuco") {
-          this.state = "PE"
-
-        } else if (this.state == "Piauí") {
-          this.state = "PI"
-
-        } else if (this.state == "Rio de Janeiro") {
-          this.state = "RJ"
-
-        } else if (this.state == "Rio Grande do Norte") {
-          this.state = "RN"
-
-        } else if (this.state == "Rio Grande do Sul") {
-          this.state = "RS"
-
-        } else if (this.state == "Rondônia") {
-          this.state = "RO"
-
-        } else if (this.state == "Roraima") {
-          this.state = "RR"
-
-        } else if (this.state == "Santa Catarina") {
-          this.state = "SC"
-
-        } else if (this.state == "São Paulo") {
-          this.state = "SP"
+    validations: {
+      type_of_address: { required },
+      BRZipCode: { required },
+      city: { required },
+      district: { required },
+      address: { required },
+      state: { required },
+    },
 
-        } else if (this.state == "Sergipe") {
-          this.state = "SE"
+    computed: {
+      type_of_address_errors () {
+        const errors = []
+        if (!this.$v.type_of_address.$dirty) return errors
+        !this.$v.type_of_address.required && errors
+          .push('type of address cant be black.!')
+        return errors
+      },
+      BRZipCode_errors () {
+        const errors = []
+        if (!this.$v.BRZipCode.$dirty) return errors
+        !this.$v.BRZipCode.required && errors.push('Zip Code cant be black.')
+        return errors
+      },
+      city_errors () {
+        const errors = []
+        if (!this.$v.city.$dirty) return errors
+        !this.$v.city.required && errors.push('City cant be black.')
+        return errors
+      },
+      state_errors () {
+        const errors = []
+        if (!this.$v.state.$dirty) return errors
+        !this.$v.state.required && errors.push('State cant be black.')
+        return errors
+      },
+      district_errors () {
+        const errors = []
+        if (!this.$v.district.$dirty) return errors
+         !this.$v.district.required && errors.push('District cant be black.')
```

Comment:
Como comentado no formulário da árvore, o texto "X cant be black" não faz muito sentido, o ideal seria "X must be filled", sendo x todos os campos que tem essa mensagem dessa forma

Reply:
Alterado. Vai no próximo commit de ajustes.

Reply:
No lugar de de "black" era para ser "blank".